hw/ip/clkmgr/lint/clkmgr.waiver

```diff
@@ -15,3 +15,12 @@ waive -rules EMPTY_PARAM_LIST -location {prim_clock_gating.sv} -regexp {.*} \
 
 waive -rules OUTPUT_NOT_DRIVEN -location {prim_clock_gating.sv} -regexp {.*} \
       -comment "Generated abstraction files do not detect drivers"
+
+waive -rules NOT_READ -location {clkmgr_reg_top.sv} -regexp {regwdata.* is not read from} \
```

Comment:
Hm, there's no signal `regwdata`? There's `reg_wdata`, but that's actually used. Anyways, the preferred way to waive unused signals is in RTL by defining a signal `unused_something = reg_wdata` (for example); this way we don't need to create waivers for each linter individually.

Reply:
o sorry you're right. It should be `reg_wdata`. You know you're right. Let me add a dummy OR in the regfile or something to cover the cases where reg_wdata / reg_be isn't fully used.